afs-network/afs-network-client/src/main/java/com/powsybl/afs/network/client/RemoteNetworkCacheService.java

```diff
@@ -65,6 +66,11 @@ private RemoteServiceConfig getConfig() {
         return Objects.requireNonNull(configSupplier.get()).orElseThrow(() -> new AfsException("Remote service config is missing"));
     }
 
+    @Override
+    public <T extends ProjectFile & ProjectCase> Network getNetwork(T projectCase, NetworkListener listener) {
+        return getNetwork(projectCase);
```

Comment:
We must at least mention in javadoc that listeners will not be added to the network.

Reply:
Is there a javadoc for powsybl-afs ?